plume-models/src/medias.rs

```diff
@@ -62,7 +62,14 @@ impl MediaCategory {
 impl Media {
     insert!(medias, NewMedia);
     get!(medias);
-    list_by!(medias, for_user, owner_id as i32);
+
+    pub fn for_user(conn: &Connection, owner: i32) -> Result<Vec<Media>> {
+        medias::table
+            .filter(medias::owner_id.eq(owner))
+            .order(medias::id.desc())
```

Comment:
is this guaranteed to be such?
do we have a create date anywhere?

Reply:
we don't have date, and the ids we use are monotonic counter, unless we overflow that 64b signed counter (not likely to happen)

Reply:
i wonder if anyone expects that editing a media pushes it to the top again…

Reply:
We can't edit media information for the moment, so…

Reply:
well then… *pushes [resolve] button*